src/dev.omaremara.bugtracker/dev/omaremara/bugtracker/model/Report.java

```diff
@@ -0,0 +1,73 @@
+package dev.omaremara.bugtracker.model;
+
+import dev.omaremara.bugtracker.model.ReportLevel;
+import dev.omaremara.bugtracker.model.ReportPriority;
+import dev.omaremara.bugtracker.model.ReportType;
+import dev.omaremara.bugtracker.model.Assigne;
+import dev.omaremara.bugtracker.model.Project;
+
+public class Report {
+    String title;
+    String description;
+    ReportLevel level;
+    ReportPriority priority;
+    ReportType type;
+    Project project;
+    Assigne assigne;
```

Comment:
`assigne` should be of type `User`.

Reply:
what do you mean User is class not enum?!

Reply:
There is no type called `assignee`, `assignee` is just a user, so this line should be `User assigne;`